packages/@okta/vuepress-site/docs/guides/registration-inline-hook/resources/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Resources
+---
+
+For full descriptions of each object included in the request and response of the Registration Inline Hook requests, see the sections [Objects in the Request from Okta](/docs/reference/registration-hook/#objects-in-the-request-from-okta) and [Objects in the Response You Send](/docs/reference/registration-hook/#objects-in-response-you-send) in the reference documentation for the Registration Inline Hook.
+
+See the following other resources for information on Inline Hooks, Registration Inline Hooks, and Self-Service Registration:
+
+* See [Okta Hooks](https://www.okta.com/hooks/) for a high-level overivew of inline hooks
```

Comment:
Wouldn't normally link to a page like that from dev docs.

Reply:
Thanks, wasn't sure. Will remove.